### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by Ibtisam34**
- [ ]  For this milestone of my portfolio website, I had implemented a popup window that includes the project details on both desktop and mobile. In order to do that, I had a need to store the information about my projects in a JavaScript object.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -13,4 +13,118 @@ closeImage.addEventListener('click', () => {
...
+      <li>${data.technology[0]}</li>
+      <li>${data.technology[1]}</li>
+      <li>${data.technology[2]}</li>
+    </ul>
+  </div>
+  <p class="title" hidden>${data.title}</p>
+  <p class="description" hidden>${data.description}</p>
+  <p class="livelink" hidden>${data.liveLink}</p>
+  <p class="sourcelink" hidden>${data.sourceLink}</p>
+  <a href="#" class="btn1" id="w2button">See Project</a>
```
Comment (leonardodiasb) [inline script.js]: - [ ] The 'See project' button in the work cards should be a `button` tag that only opens the popup. Using the `a` tag is redirecting the user to the top of the page making a bad user experience. Kindly change that.

File: style.css [hunk 2]
```diff
@@ -124,6 +124,142 @@ header {
...
+  background-color: #fff;
+  justify-content: center;
+  padding: 45px 10px 30px 10px;
+  transition: visibility 0s ease 0.5s;
+}
+
+.Recentwork1.active {
+  visibility: visible;
+  opacity: 1;
+}
```
Comment (leonardodiasb) [inline style.css]: - [ ] Your popup window covers the entire page, not creating the feeling of a popup window, and this way the user can misunderstand and think that he/she was redirected to another page. Kindly add margins to the popup and add some background filter with some dark overlay so your popup looks like it is floating above the content of the page.
 -Your design:
Mobile:
image
Desktop:
image


-Figma template:
Mobile:
image

Desktop:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Ibtisam34, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of GitHub flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -644,15 +825,73 @@ textarea {
...
+
+  .css-display {
+    display: none;
+  }
+
+  .css-display.show {
+    display: block;
+  }
+
   .Recentwork hr {
```
Comment (Santiago220991) [inline style.css:858]: I think you did a great job implementing the changes suggested by the last code reviewer. However, there are still issues to fix. Kindly make sure your popup windows are being rendered correctly as the last [code reviewer suggested](https://github.com/Ibtisam34/portofolio-mobile/pull/11#discussion_r999294966). 
- The popup windows should be centered on both versions mobile and desktop. Kindly fix this. :pray: :pray: 

Your design
image
image

Figma Template
image

image

- The popup window on the desktop version has an issue when is rendering. The popup window has a kind of transparency making the background appear inside the popup windows. Check the image below, please. :eyes: :eyes: 
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @Ibtisam34, 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (Ibtisam34) [reply style.css:858]: @Santiago220991, Please I have made the necessary adjustments, help me check through before the final submission

**Comment by Ibtisam34** [comment]
@Santiago220991, I had added the button kindly check it and give me your feedback  as soon as your possible

**Review by MahmoudBakr23 — APPROVED** [review]
Hi team😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done working this hard on the required changes from the previous reviewers. I see you have centered the popup menu on both desktop and mobile screens. 👍 
- Excellent communication with the reviewers

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**